src/app/pdok/lookup-object/lookup-object.component.scss

```diff
@@ -19,4 +19,8 @@ mat-form-field {
     &.show {
         display: block;
     }
+}
+
+.open_source {
```

Comment:
shouldn't the className be 'open-source' ? open_source indicates that there is also a class 'open' in the file and that's not the case